Jenkinsfile

```diff
@@ -264,7 +264,8 @@ def runBuild(abiFilter, instrumentationTestTarget) {
   if (releaseBranches.contains(currentBranch)) {
     stage('Publish to OJO') {
       withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'bintray', passwordVariable: 'BINTRAY_KEY', usernameVariable: 'BINTRAY_USER']]) {
-        sh "chmod +x gradlew && ./gradlew -PbintrayUser=${env.BINTRAY_USER} -PbintrayKey=${env.BINTRAY_KEY} assemble ojoUpload --stacktrace"
+        sh "chmod +x gradlew && ./gradlew assemble --stacktrace"
```

Comment:
We should already have assembled everything as part of the build phase. So I don't think this is needed.

Reply:
It is not needed 👌